src/test/java/tests/UsersTest.java

```diff
@@ -124,4 +125,98 @@ public void youngerThanParameterTest() {
                 () -> Assertions.assertFalse(usersResponse.getBody().isEmpty(), "Response body does not contain user"),
                 () -> Assertions.assertTrue(usersResponse.getBody().stream().allMatch(user -> user.getAge() < 19), "User age is not younger than than target age"));
     }
+
+    @Test
+    public void putUserTest() {
+        ResponseEntity<List<User>> usersResponse = client.getUsers();
```

Comment:
create a separate class ex.: UpdateUserTest and move all the tests of the task there. In `@BeforeEach` create new userToUpdate using for ex. `userClient.createAvailableUser(new User(...))`. Then in the tests you will use this user and will create new user according to the test task